paddlenlp/transformers/model_utils.py

```diff
@@ -270,6 +270,16 @@ def from_pretrained(cls, pretrained_model_name_or_path, *args, **kwargs):
                 init_kwargs = json.load(f)
         else:
             init_kwargs = init_configuration
+
+            model_config_file_path = os.path.join(default_root,
+                                                  cls.model_config_file)
+            # check if there is model config file in cache directory
+            if pretrained_model_name_or_path in cls.pretrained_init_configuration and init_kwargs is not None and not os.path.exists(
```

Comment:
I have add the checking situation:

```python
init_kwargs is not None
````

So, I'm curious about that whether model_config_file and init_kwargs will be None at the same time ? I think this will be boundary case which can be avoided by developers although.